addon/content/attachments.jsx

```diff
@@ -2,8 +2,10 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 
-/* globals PropTypes, React, SvgIcon, attachmentActions */
-/* exported Attachments */
+import React from "react";
+import PropTypes from "prop-types";
+import { attachmentActions } from "./reducer-attachments.js";
+import { SvgIcon } from "./svgIcon.jsx";
```

Comment:
Is there a particular reason you're keeping file extensions in? Since native modules cannot import `jsx`, I don't see a benefit in keeping the extension.

Reply:
Without the file extension webpack / whatever can't find the file, I think it treats it as something like a node_modules import.

```
Module not found: Error: Can't resolve './compose' in '/Users/mark/dev/thunderbird-conversations/addon/compose'
Did you mean 'compose.jsx'?
BREAKING CHANGE: The request './compose' failed to resolve only because it was resolved as fully specified
(probably because the origin is a '*.mjs' file or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
```

Reply:
Very strange...I see this reference https://github.com/babel/babel/issues/8462 which says it's resolved in `@babel/runtime@7.12.0`. It seems like `babel.config.cjs` is still saying `modules: false`. Is this on purpose? Doesn't that mean things aren't bundled?

Reply:
`modules: false` is for the type of output that Babel has, in this case false is keep them as es6 modules. I've currently removed that here, as now we're using Webpack, I understand that it lets Babel work it out automatically, which is better for Webpack.